aws/cloudformation/scylla.yaml.j2

```diff
@@ -10,37 +10,76 @@ Description: >-
 
   Caution: password authentication isn't enabled by default
 
+Metadata:
+  AWS::CloudFormation::Interface:
+    ParameterGroups:
+      - Label:
+          default: Scylla Parameters
+        Parameters:
+          - ScyllaClusterName
+          - ScyllaAmi
+          - ScyllaSeedIPs
+      - Label:
+          default: AWS Parameters
+        Parameters:
+          - InstanceType
+          - InstanceCount
+          - AvailabilityZone
+          - CIDR
+          - EnablePublicAccess
+          - SSHLocation
+          - KeyName
+    ParameterLabels:
+      ScyllaClusterName:
+        default: Scylla cluster name
+      ScyllaAmi:
+        default: Scylla AMI for the EC2 instances
+      ScyllaSeedIPs:
+        default: Scylla seed nodes IPs
+      InstanceType:
+        default: EC2 instance type
+      InstanceCount:
+        default: Number of Scylla nodes (EC2 Instances)
+      AvailabilityZone:
+        default: Availability Zone
+      CIDR:
+        default: CIDR block for Scylla VPC
+      EnablePublicAccess:
+        default: Allow public access (SSH)
+      SSHLocation:
+        default: Allowed subnet for public access (SSH)
+
 Parameters:
-  ClusterName:
+  ScyllaClusterName:
     Type: String
 
-  PublicCidrAccess:
+  SSHLocation:
```

Comment:
Why Location?

Reply:
No good answer + it shouldn't be service specific (SSH in this case),
Changed to `PublicAccessCIDR`